### PR TITLE
[WIP] Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
 /kind enhancement

**What this PR does / why we need it**:
This PR migrates the CI from Travis to GitHub Actions.

**Which issue(s) this PR fixes**:
Fixes #

**Test Report Added?**:
/kind TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: SODA Multicloud CI Build
+
+on:
+  push:
+    branches: [ master ]
```
Comment (kumarashit) [inline .github/workflows/ci.yml:5]: why to restrict on branches? why not on: [push, pull_request]..allow for all

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        wget -qO - https://packages.confluent.io/deb/5.3/archive.key | sudo apt-key add -
+        wget -q -O- "https://download.ceph.com/keys/release.asc" | sudo apt-key add -
+        sudo add-apt-repository "deb [arch=amd64] https://packages.confluent.io/deb/5.3 stable main"
+        sudo apt-add-repository "deb https://download.ceph.com/debian-luminous/ xenial main"
+        sudo apt-get update && sudo apt-get install -y libradosstriper-dev librados-dev librdkafka-dev
+      
+    - name: Build the multicloud docker images
+      run: |
+        make docker
+        docker-compose up -d
```
Comment (kumarashit) [inline .github/workflows/ci.yml:44]: you can remove debug build

File: .github/workflows/ci.yml (hunk 1 above)
Comment (anvithks) [reply .github/workflows/ci.yml:5]: This is still WIP. restricted for now. Will make this change.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (anvithks) [reply .github/workflows/ci.yml:44]: Could you please elaborate? This step builds the docker images using the latest source code that is checked out in the previous steps.

Comment (kumarashit) [reply .github/workflows/ci.yml:44]: Sorry commented at wrong place/wrong PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kumarashit) [reply .github/workflows/ci.yml:5]: ok

**Comment by anvithks** [comment]
Closing this for now. will raise the PR with the appropriate repository permissions.

**Pull request closed without merging**